tests/gis_tests/geos_tests/test_io.py

```diff
@@ -90,9 +97,13 @@ def test04_wkbwriter(self):
 
         # Ensuring bad byteorders are not accepted.
         for bad_byteorder in (-1, 2, 523, "foo", None):
-            # Equivalent of `wkb_w.byteorder = bad_byteorder`
-            with self.assertRaises(ValueError):
-                wkb_w._set_byteorder(bad_byteorder)
+            with self.subTest(bad_byteorder=bad_byteorder):
```

Comment:
etc

Reply:
I nested the `with` calls when appropriate.